app/controllers/boardController.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const db = require('../models');
+const { validationResult } = require('express-validator');
+
+exports.getNewPage = (req, res) => {
+    db.Users.findOne({
+        where: {id: req.user.id}
+    })
+        .then((user) => {
+            res.render('new', {
+                input: '',
+                user: user.name,
+                error: ''
+            });
+        });
+};
+
+exports.newBoard = async (req, res, next) => {
+    const errors = validationResult(req);
+    if(!errors.isEmpty()) {
+        return res.render('new', { 
+            input: req.body,
+            user: req.user.name,
+            error: errors.array()[0].msg
+        });
+    }
+    const newBoard = await db.Boards.build({
+        title: req.body.title,
+        text: req.body.text,
+        userId: req.user.id,
+        userName: req.user.name,
+        like: 0
+    });
+    newBoard
+        .save()
+            .then(() => {
```

Comment:
ここのインデントは不要です。
他も同様です。
```suggestion
        .save()
        .then(() => {
```